spec/lib/solidus_graphql_api/context_spec.rb

```diff
@@ -103,6 +108,21 @@
     it { is_expected.to eq default_store }
   end
 
+  describe '#current_pricing_options' do
+    subject { schema_context.current_pricing_options }
+
+    let(:default_currency) { 'EUR' }
+    let(:country_iso) { 'IT-it' }
+    let!(:default_store) do
+      create(:store, default: true, default_currency: default_currency, cart_tax_country_iso: country_iso)
+    end
+
+    after { subject }
```

Comment:
With this hook we call the subject two times in the next spec, can we avoid it by changing the last spec?